uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -200,7 +200,9 @@ define([
 
         $scope.$broadcast('ct-spinner-show');
 
-        documentStatus = vm.isRole('admin') ? '3' : (vm.files.length || vm.uploader.queue.length ? '2' : '1');
+        documentStatus = vm.isRole('admin') ?
+          (vm.files.length || vm.uploader.queue.length ? '3' : '1') :
```

Comment:
Please dont use static numbers like 3 or 1, its impossible to know what they are. You must be getting all the status's from backend. So you can create a function called `getStatusIdByName()`, this function searches for the ID given the name, so you can use `(vm.files.length || vm.uploader.queue.length ?  getStatusIdByName('<>'): getStatusIdByName('<>')) :`